src/Components/AddVideoToPlayList/PlayListNames.js

```diff
@@ -1,64 +1,60 @@
+import { callServer } from "../../API/api_call";
+import { useAuth } from "../../context/auth-context";
 import { usePlaylistModal } from "../../context/playlist-context";
 import { usePlaylist } from "../../context/video-context";
 
 const PlaylistNameList = ({ video }) => {
   const { state, dispatch } = usePlaylist();
   const { modalDispatch, manageToast } = usePlaylistModal();
-  const PLAYLIST_NAMES = Object.keys(state.playLists);
-  const addVideoToPlayList = (video, playListName) =>
-    state.videolist.map((vid) =>
-      vid.id === video.id
-        ? {
-            ...vid,
-            playlists: !vid.playlists.find(
-              (playlist) => playListName === playlist
-            )
-              ? [...vid.playlists, playListName]
-              : vid.playlists.filter((playlist) => playlist !== playListName),
-          }
-        : vid
-    );
+  const {
+    authState: { userId },
+  } = useAuth();
 
-  const managePlaylistAndVideoList = (playlist) => {
-    dispatch({
-      type: "ADDED_VIDEO_TO_PLAYLIST",
-      value: {
-        videos: addVideoToPlayList(video, playlist),
-        playlistname: playlist,
-      },
-    });
-    dispatch({
-      type: "SELECTED_PLAYLIST",
-      value: { playlistName: playlist, video: video },
-    });
-  };
-  const isPlaylistCheckboxChecked = (playlist) =>
-    video?.playlists.find((playlst) => playlst === playlist) === undefined
+  const PLAYLIST = state.playLists;
```

Comment:
Why PLAYLIST binding is capital ?

Reply:
I remembered const variables are defined in Capital letters